src/components/ConfigureGame/ConfigureGame.js

```diff
@@ -39,56 +33,54 @@ const ConfigureGame = () => {
     }
   };
 
-  const handleNameChange = (playerName, playerIndex) => {
-    const newPlayers = [...players];
-    newPlayers[playerIndex].name = playerName;
-    setPlayers(newPlayers);
-  };
-
-  const updatePlayerAvatar = (newAvatar, playerIndex) => {
-    const newData = [...players];
-    newData[playerIndex].avatar = newAvatar;
-    setPlayers(newData);
-  };
-
   return (
     <div className="configure-screen">
       {players.map((player, index) => {
         return (
           <div className="configure-screen__name-input-wrapper" key={player.id}>
-            <AvatarPicker
-              avatar={player.avatar}
-              onChange={newAvatar => updatePlayerAvatar(newAvatar, index)}
-            />
-            <Input
-              onChange={event => handleNameChange(event.target.value, index)}
-              placeholder="Enter player's name"
-              value={player.name}
-            />
-            {numberOfPlayers > MIN_PLAYERS && (
-              <Button
-                styleReset
-                onClick={() => removeInput(index)}
-                aria-label="Remove player"
-              >
-                -
-              </Button>
-            )}
+            <div className="configure-screen__player-info">
```

Comment:
nit: base all of these BEM names with `configure-game` since that is the name of the component?